test/scenarios-periodics/el92-src@offline.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart-offline.ks.template
```

Comment:
testing locally fails b/c the 3rd arg is undefined

Reply:
Thank you, fixing asap.